autoscheduler/frontend/src/components/SchedulingPage/CourseSelectColumn/CourseSelectCard/ExpandedCourseCard/SectionSelect/SectionSelect.css

```diff
@@ -184,6 +184,101 @@
     transform: rotate3d(1, 0, 0, -180deg);
 }
 
+/* Former Basic Select */
+.fit-content, .table-container, .select-root {
+    width: -moz-fit-content;
+    width: fit-content;
+}
+
+.select-root:global(.MuiOutlinedInput-input) {
+    padding: 8px 14px;
+}
+
+.select-menu:global(.MuiSelect-selectMenu.MuiSelect-selectMenu) {
+    padding-right: 30px;
+}
+
+.placeholder-text {
+    padding-bottom: 8px;
+    padding-right: 8px;
+}
+
+/* filter stuff */
+.sub-title {
+    margin-top: 6px !important;
+}
+.sub-title.error-text {
+    margin-bottom: 12px !important;
+}
+/* we want to select first of class, this is currently the only way to do it */
+/* first */
+.table-container {
+    /* width: calc(100% - 9px); */
+    width: 100%;
+    padding-top: 6px;
+}
+/* all others */
+.table-container ~ .table-container {
+    padding-top: 0;
+    /* width: 100%; */
+}
+.table-container:last-of-type {
+    padding-bottom: 6px;
+    overflow: hidden;
+}
+.table-container table {
+    flex-grow: 1;
+}
+.table-container table tr td:last-of-type {
+    text-align: right;
+}
+.table-container .accordian-root {
+    /* width: 100%; */
+    box-shadow: none !important;
+}
+.table-container :global(.MuiCollapse-wrapper) {
+    width: 100%;
+}
+.table-container :global(#show-hide-section) {
+    height: 100%;
+    display: flex;
+    flex-direction: row;
+}
+.table-container :global(#show-hide-filters) {
+    padding-right: 9px;
+}
+.table-container .accordian-summary {
+    padding-left: 0;
+    padding-right: 9px;
+    margin: 0;
+}
+.table-container .accordian-summary .MuiExpansionPanelSummary-content {
+    margin: 0;
+}
+.table-container .accordian-details {
+    margin-top: -10px;
+    padding-bottom: 4px;
+    padding: 8px 0 0 4px;
+    height: 100%;
+    width: calc(100% - 4px);
+}
+.table-container .button-root {
+    width: calc(100% - 5px);
+    text-align: left;
+    display: inline-flex;
+    flex-direction: row;
+}
+/* sections stuff */
+.table-container .accordian-details .sections-wrapper {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    height: fit-content;
```

Comment:
So this line is what fixes the scroll bar issue, but it also causes transitions to bug out?